app/src/main/java/com/project/gamedb/data/source/local/LocalRepository.kt

```diff
@@ -6,17 +6,7 @@ import com.project.gamedb.data.source.remote.OnDataLoadedCallback
 
 class LocalRepository private constructor(private val local: GameDataSource.Local) :
```

Comment:
Sao lại LocalRepository 
```suggestion
class GameLocalRepository private constructor(private val local: GameDataSource.Local) :
```